src/zkp/pisch.rs

```diff
@@ -200,18 +205,20 @@ impl PiSchProof {
     }
     pub(crate) fn from_message(message: &Message) -> Result<Self> {
         message.check_type(MessageType::Keygen(KeygenMessageType::R3Proof))?;
-        let keygen_decommit: PiSchProof = deserialize!(&message.unverified_bytes)?;
-        Ok(keygen_decommit)
+        let pischproofbytes: PiSchProof = deserialize!(&message.unverified_bytes)?;
```

Comment:
minor name quibble: if you keep this name, it should be snake case, like `pisch_proof_bytes`. I would actually remove the `_bytes` modifier, though, since the type of this variable is a full-on `PiSchProof`, not just the bytes that can become a proof.